configs/arrow.json

```diff
@@ -3,21 +3,15 @@
   "start_urls": ["https://arrow-kt.io"],
   "stop_urls": [
     "https://arrow-kt.io/docs/[0-9]",
-    "https://arrow-kt.io/docs/next",
-    ".*(?<!index\\.html)$"
+    "https://arrow-kt.io/docs/next"
   ],
   "sitemap_urls": ["https://arrow-kt.io/sitemap.xml"],
   "selectors": {
     "default": {
-      "lvl0": {
-        "selector": ".brand-title",
-        "global": true,
-        "default_value": "Documentation"
-      },
-      "lvl1": ".doc-content h1",
```

Comment:
Do you have any query examples where it doesn't work?

Replacing this selector would add a lot of noise to the results I think

|before|after|
|--|--|
|<img width="624" alt="Screenshot 2021-04-05 at 15 09 33" src="https://user-images.githubusercontent.com/20689156/113577253-40316c00-9621-11eb-9300-c233a1acb6b4.png">|<img width="643" alt="Screenshot 2021-04-05 at 15 11 24" src="https://user-images.githubusercontent.com/20689156/113577275-4889a700-9621-11eb-9da2-19934de48e9e.png">|

Reply:
Undone, thanks!!